.github/workflows/sql-test-workflow.yml

```diff
@@ -0,0 +1,104 @@
+name: SQL Plugin Tests
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'dev-*'
+    paths:
+      - '**/*.java'
+      - '**/*.g4'
+      - '!sql-jdbc/**'
+      - '**gradle*'
+      - '**lombok*'
+      - '**checkstyle*'
+      - 'integ-test/**'
+      - 'docs/**'
+      - 'doctest/**'
+      - '**/*.jar'
+      - '**/*.pom'
+      - '.github/workflows/sql-test-workflow.yml'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        java:
+          - 11
+          - 17
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Set up JDK ${{ matrix.java }}
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: ${{ matrix.java }}
+    
+    - name: Run tests
+      id: tests
+      run: |
+        # checkstyle
```

Comment:
Is this unpacking the build target? Just realized that whatever task that used to be, running it using `./gradlew --continue <task>` will keep running it after first failure.

Could you try that and see if it does what you are looking for? Sorry I didn't realize this earlier.

Reply:
`--continue` says to ignore all failures and keep running.
Below is the output from `./gradlew --continue build`.
1. Too many garbage in the output. We have to parse it to provide a brief report.
2. Tasks executed in alphabetical order - `:core:x`, then `doctest`, `integ-test`, and so on, with `ppl` in the very end.

```
FAILURE: Build completed with 5 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':core:checkstyleMain'.
> Checkstyle rule violations were found. See the report at: file:///mnt/c/GitHub/opensearch-project-sql-bq/core/build/reports/checkstyle/main.html
  Checkstyle files with violations: 1
  Checkstyle violations by severity: [error:1]


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':core:checkstyleTest'.
> Checkstyle rule violations were found. See the report at: file:///mnt/c/GitHub/opensearch-project-sql-bq/core/build/reports/checkstyle/test.html
  Checkstyle files with violations: 1
  Checkstyle violations by severity: [error:1]


* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

3: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':core:jacocoTestCoverageVerification'.
> Rule violated for class org.opensearch.sql.expression.datetime.DateTimeFunction: lines covered ratio is 0.9, but expected minimum is 1.0
  Rule violated for class org.opensearch.sql.expression.datetime.DateTimeFunction: branches covered ratio is 0.9, but expected minimum is 1.0

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

4: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':doctest:startOpenSearch'.
> The process terminated unexpectedly - status code 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

5: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':integ-test:integTest'.
> There were failing tests. See the report at: file:///mnt/c/GitHub/opensearch-project-sql-bq/integ-test/build/reports/tests/integTest/index.html

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================
```